app/src/services/dataService.js

```diff
@@ -249,12 +255,18 @@ module.exports = {
         ipcPlanId = ipcPlanObj.ipcPlanId;
 
         await db.Business.create({...business, ipcPlanId: ipcPlanId}, {transaction: t});
-        await db.Location.create({...location, ipcPlanId: ipcPlanId}, {transaction: t});
+
+        const locationObj = await db.Location.create({...location, ipcPlanId: ipcPlanId}, {transaction: t});
+        let locationId = locationObj.locationId;
 
         await db.Contact.bulkCreate(contacts.map(c => {
           return {...c, ipcPlanId: ipcPlanId};
         }), {transaction: t});
 
+        await db.MapLocation.bulkCreate(mapLocations.map(ml => {
+          return {...ml, locationId: locationId};
```

Comment:
As you only use locationId once in this function, you can streamline this directly into line 267 and drop the let locationId on line 260.
```js
return {...ml, locationId: locationObj.locationId};
```